srcjs/cyto/cytoStyle.js

```diff
@@ -82,7 +83,7 @@ let graphStyles = {
       "shape-polygon-points": nodeShapes.end.shape,
       width: nodeShapes.end.width,
       height: nodeShapes.end.height,
-      "text-max-width": maxTextWidth,
+      "text-max-width": `${labelWidth * 1.5}px`,
```

Comment:
What's the 1.5 factor for?

Reply:
The ending area usually has long autogenerated labels.  Having it be a little bigger doesn't hurt.  (Tony approved the size bump).